globalbibletools/lexicon-translation
# feat: project management tools

## What Has Changed:
- Added new command `createNewProject`
- Created new `languages.ts` file (sourced from [here](https://iso639-3.sil.org/sites/iso639-3/files/downloads/iso-639-3.tab))
## Connected Issues:
partially solves #5 
## QA Steps
1. Run the extension.
2. Run the "Create New Project" command
3. Ensure that all the necessary information is gathered for the project.
4. Ensure that (eventually) the project is created with:
- The proper metadata (see the [scripture burrito docs](https://scripture-burrito.readthedocs.io/))
- The source and target entries.
